package.json

```diff
@@ -12,12 +12,13 @@
     "eslint-publish-release": "./bin/eslint-publish-release.js"
   },
   "scripts": {
-    "test": "mocha tests/lib/*.js",
     "lint": "eslint .",
-    "generate-alpharelease": "node ./bin/eslint-generate-prerelease.js alpha",
-    "generate-betarelease": "node ./bin/eslint-generate-prerelease.js beta",
-    "generate-release": "node ./bin/eslint-generate-release.js",
-    "publish-release": "node ./bin/eslint-publish-release.js"
+    "release:generate:latest": "eslint-generate-release",
+    "release:generate:alpha": "eslint-generate-prerelease alpha",
+    "release:generate:beta": "eslint-generate-prerelease beta",
+    "release:generate:rc": "eslint-generate-prerelease rc",
+    "release:publish": "eslint-publish-release",
```

Comment:
These commands don't seem to work, for example:

```
> eslint-release@3.2.0 release:generate:latest
> eslint-generate-release

'eslint-generate-release' is not recognized as an internal or external command,
operable program or batch file.
```

I did `npm install` but that doesn't seem to handle `bin` from the package itself.

Reply:
>I did npm install but that doesn't seem to handle bin from the package itself.

I think you will need to run the `npm link` command as well. But I'll revert it to the old behavior.